src/entity/Settings.ts

```diff
@@ -54,6 +54,9 @@ export class Settings {
   @Column({ default: true })
   autoDismissNotifications: boolean;
 
+  @Column({ type: 'text', default: null })
```

Comment:
Our default value should be `header`.
```suggestion
  @Column({ type: 'text', default: 'header` })
```